### PR TITLE
Feature - embolden matches

### PR DESCRIPTION
**Pull request by joefitter**
* Added component to embolden exact matches from search query

### CODE REVIEW

File: ui/components/Dataset/index.js [hunk 1]
```diff
@@ -9,6 +9,13 @@ import { useQueryContext } from '../../context/query';
 
 const DATE_FORMAT = 'do MMM yyyy';
 
+function Embolden({ children }) {
+  const { getSelections } = useQueryContext();
+  const { q } = getSelections();
+  const re = new RegExp(`(${q})`, 'ig')
+  return <span dangerouslySetInnerHTML={{ __html: children.replace(re, '<b>$1</b>') }} />
```
Comment (lennym) [inline ui/components/Dataset/index.js:16]: Given that this is being set with user input from the query string, I think this is _actually_ dangerous.

Comment (lennym) [reply ui/components/Dataset/index.js:16]: `?q=<script>alert('All your base are belong to me');</script>`

Comment (roc) [reply ui/components/Dataset/index.js:16]: Just to have a record of this on here: we've concluded this is _probably okay_ given we're only replacing content that exists in data if it matches, so malicious injections won't be matched. We can investigate more at pen test

**Review by roc — CHANGES_REQUESTED** [review]
* package-lock in the root dir should be removed
* strong tag instead of <b> for maximum WHATCG web semantics 2008 edition markup

**Review by roc — APPROVED** [review]

File: ui/components/Pagination/style.module.scss [hunk 2]
```diff
@@ -27,8 +27,10 @@
     text-decoration: none;
     padding: $nhsuk-gutter / 2 $nhsuk-gutter $nhsuk-gutter / 2 $nhsuk-gutter;
 
-    &:hover, &:focus {
+    &:hover,
+    &:focus {
       outline: 3px solid $yellow;
+      cursor: pointer;
```
Comment (lennym) [inline ui/components/Pagination/style.module.scss:33]: Actually adding the query string to the `href` would be better.

**Pull request merged**